search-engines.lisp

```diff
@@ -622,6 +622,243 @@ REQUEST-ARGS is a list of args to pass to request function."
+                            (:large "isz:l")
+                            (:medium "isz:m")
+                            (:large "isz:l")
+                            (:icon "isz:i")))
+  (images-size-predefined "image-size-select" ((:400x300 "isz:lt,islt:qsvgs")
+                                               (:640x480 "isz:lt,islt:vga")
+                                               (:800x600 "isz:lt,islt:svga")
+                                               (:1024x768 "isz:lt,islt:xga")
+                                               (:1600x1200 "isz:lt,islt:2mp") ; 2MP
+                                               (:2272x1704 "isz:lt,islt:4mp") ; 4MP
+                                               (:2816x2112 "isz:lt,islt:6mp") ; 6MP
+                                               (:3264x2448 "isz:lt,islt:8mp") ; 8MP
+                                               (:3648x2736 "isz:lt,islt:10mp") ; 10MP
+                                               (:4096x3072 "isz:lt,islt:12mp") ; 12MP
+                                               (:4480x3360 "isz:lt,islt:15mp") ; 15MP
+                                               (:5120x3840 "isz:lt,islt:20mp") ; 20MP
+                                               (:7216x5412 "isz:lt,islt:40mp") ; 40MP
+                                               (:9600x7200 "isz:lt,islt:70mp"))) ; 70MP
+  (images-size-exact-width "flimgexwidth" (:function #'startpage-image-size))
+  (images-size-exact-height "flimgexheight" (:function #'startpage-image-size))
+  (images-color "flimgcolor" ((:any "ic:")
+                              (:color-only "ic:color")
+                              (:black-white "ic:gray")
+                              (:transparent "ic:trans")
+                              (:red "ic:specific,isc:red")
+                              (:orange "ic:specific,isc:orange")
+                              (:yellow "ic:specific,isc:yellow")
+                              (:green "ic:specific,isc:green")
+                              (:teal "ic:specific,isc:teal")
+                              (:blue "ic:specific,isc:blue")
+                              (:purple "ic:specific,isc:purple")
+                              (:pink "ic:specific,isc:pink")
+                              (:gray "ic:specific,isc:gray")
+                              (:black "ic:specific,isc:black")
+                              (:brown "ic:specific,isc:brown")))
+  (images-type "flimgtype" ((:any "")
+                            (:jpg "jpg")
+                            (:png "png")
+                            (:gif "gif")))
+  (videos-filter "sort_by" ((:relevant "")
+                            (:popular "popular")
+                            (:recent "recent")))
+  (videos-length "with_duration" ((:any "")
+                                  (:short "short")
+                                  (:medium "medium")
+                                  (:long "long")))
+  (news-date "with_date" ((:day "d")
```

Comment:
Is news sorted by date by default? Maybe have `(:any "")` here?

Reply:
Done.